### ISSUE TITLE
TypeError: getProcessedPlugins is not a function

### ISSUE THREAD

**Issue by devunion** [jammeryhq/gridsome-starter-articles#2]
Tried to set up new project from the latest template. Node version: 12.20.0. Output log: 

```
C:\work\vitya\my_projects\ContentMaster\my-site>gridsome develop
Gridsome v0.7.23

Initializing plugins...
Tag > Failed to add node: Duplicate key for property id: tag1
Tag > Failed to add node: Duplicate key for property id: tag2
Tag > Failed to add node: Duplicate key for property id: tag3
Tag > Failed to add node: Duplicate key for property id: tag4
Tag > Failed to add node: Duplicate key for property id: tag5
Load sources - 0.21s
Create GraphQL schema - 0.2s
Create pages and templates - 0.09s
Generate temporary code - 0.05s
i Added 10 nodes to Search Index                                                                                                                                                                   gridsome-plugin-flexsearch 14:24:50
FlexSearchIndex - 0.02s
Bootstrap finish - 2.04s
i Serving search index...                                                                                                                                                                          gridsome-plugin-flexsearch 14:24:51
63% building 447/483 modules 36 active ...ntentMaster\my-site\node_modules\vue-loader\lib\index.js??vue-loader-options!C:\work\vitya\my_projects\ContentMaster\my-site\src\layouts\partials\ResponsiveMenu.vue?vue&type=script&lang=js&c
ode block or inline code language not specified in markdown. applying generic code block


 ERROR  Failed to compile with 5 errors                                                                                                                                                                                         14:24:59


 error  in ./node_modules/prismjs/plugins/line-numbers/prism-line-numbers.css

Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: PostCSS plugin postcss-nested requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users
    at Processor.normalize (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\processor.js:167:15)
    at new Processor (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\processor.js:56:25)
    at postcss (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\postcss.js:55:10)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\util\parseObjectStyles.js:24:33
    at arrayMap (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\lodash\lodash.js:639:23)
    at map (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\lodash\lodash.js:9580:14)
    at Function.flatMap (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\lodash\lodash.js:9283:26)
    at parseObjectStyles (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\util\parseObjectStyles.js:23:26)
    at parseObjectStyles (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\util\parseObjectStyles.js:20:12)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\util\processPlugins.js:37:123
    at arrayMap (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\lodash\lodash.js:639:23)
    at map (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\lodash\lodash.js:9580:14)
    at Function.flatMap (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\lodash\lodash.js:9283:26)
    at parseStyles (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\util\processPlugins.js:37:26)
    at addComponents (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\util\processPlugins.js:131:18)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\plugins\container.js:101:5

 @ ./node_modules/prismjs/plugins/line-numbers/prism-line-numbers.css 4:14-156 14:3-18:5 15:22-164
 @ ./src/main.js
 @ ./node_modules/gridsome/app/app.js
 @ ./node_modules/gridsome/app/entry.client.js
 @ multi webpack/hot/dev-server webpack-hot-middleware/client?name=app&reload=true&noInfo=true ./node_modules/gridsome/app/entry.client.js ./node_modules/gridsome/app/entry.sockjs.js

 error  in ./node_modules/prismjs/themes/prism.css

Module build failed (from ./node_modules/postcss-loader/src/index.js):
TypeError: getProcessedPlugins is not a function
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\processTailwindFeatures.js:75:83
    at LazyResult.run (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:288:14)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:212:26)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:225:14)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:217:17

 @ ./node_modules/prismjs/themes/prism.css 4:14-137 14:3-18:5 15:22-145
 @ ./src/main.js
 @ ./node_modules/gridsome/app/app.js
 @ ./node_modules/gridsome/app/entry.client.js
 @ multi webpack/hot/dev-server webpack-hot-middleware/client?name=app&reload=true&noInfo=true ./node_modules/gridsome/app/entry.client.js ./node_modules/gridsome/app/entry.sockjs.js

 error  in ./node_modules/tailwindcss/tailwind.css

Module build failed (from ./node_modules/postcss-loader/src/index.js):
TypeError: getProcessedPlugins is not a function
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\processTailwindFeatures.js:75:83
    at LazyResult.run (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:288:14)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:212:26)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:225:14)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:217:17

 @ ./node_modules/tailwindcss/tailwind.css 4:14-134 14:3-18:5 15:22-142
 @ ./node_modules/gridsome-plugin-tailwindcss/gridsome.client.js
 @ ./src/.temp/plugins-server.js
 @ ./node_modules/gridsome/app/app.js
 @ ./node_modules/gridsome/app/entry.client.js
 @ multi webpack/hot/dev-server webpack-hot-middleware/client?name=app&reload=true&noInfo=true ./node_modules/gridsome/app/entry.client.js ./node_modules/gridsome/app/entry.sockjs.js

 error  in ./src/pages/404.vue?vue&type=style&index=0&id=d989eb38&scoped=true&lang=css&

Module build failed (from ./node_modules/postcss-loader/src/index.js):
TypeError: getProcessedPlugins is not a function
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\processTailwindFeatures.js:75:83
    at LazyResult.run (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:288:14)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:212:26)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:225:14)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:217:17

 @ ./node_modules/vue-style-loader!./node_modules/css-loader/dist/cjs.js??ref--2-oneOf-1-1!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/postcss-loader/src??ref--2-oneOf-1-2!./node_modules/cache-loader/dist
/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/pages/404.vue?vue&type=style&index=0&id=d989eb38&scoped=true&lang=css& 4:14-396 14:3-18:5 15:22-404
 @ ./src/pages/404.vue?vue&type=style&index=0&id=d989eb38&scoped=true&lang=css&
 @ ./src/pages/404.vue
 @ ./src/.temp/routes.js
 @ ./node_modules/gridsome/app/router.js
 @ ./node_modules/gridsome/app/entry.sockjs.js
 @ multi webpack/hot/dev-server webpack-hot-middleware/client?name=app&reload=true&noInfo=true ./node_modules/gridsome/app/entry.client.js ./node_modules/gridsome/app/entry.sockjs.js

 error  in ./src/assets/scss/main.scss

Module build failed (from ./node_modules/postcss-loader/src/index.js):
TypeError: getProcessedPlugins is not a function
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\tailwindcss\lib\processTailwindFeatures.js:75:83
    at LazyResult.run (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:288:14)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:212:26)
    at LazyResult.asyncTick (C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:225:14)
    at C:\work\vitya\my_projects\ContentMaster\my-site\node_modules\postcss\lib\lazy-result.js:217:17

 @ ./src/assets/scss/main.scss 4:14-215 14:3-18:5 15:22-223
 @ ./src/main.js
 @ ./node_modules/gridsome/app/app.js
 @ ./node_modules/gridsome/app/entry.client.js
 @ multi webpack/hot/dev-server webpack-hot-middleware/client?name=app&reload=true&noInfo=true ./node_modules/gridsome/app/entry.client.js ./node_modules/gridsome/app/entry.sockjs.js
```

**Comment by noxify** [comment]
Based on this error
> Error: PostCSS plugin postcss-nested requires PostCSS 8.

i think this will be solved with #1 

I will do this in the next days.

**Comment by devunion** [comment]
@noxify yes, just provided some additional details which can be helpful.
